yarn.lock

```diff
@@ -1035,16 +1046,52 @@
   optionalDependencies:
     sharp-cli "1.10.0"
 
-"@expo/json-file@^8.1.3":
-  version "8.1.3"
-  resolved "https://registry.yarnpkg.com/@expo/json-file/-/json-file-8.1.3.tgz#75d265d57facb85f52079031a53c7c9f5fc6c50b"
-  integrity sha512-2Jjn/gN75n526lWD1oIOQvQTQt3UMbICjeDgO3Tjr48zkWPYCakWBbS2cj3UnBIUqd30vIkGsp4fRh2JFEoocA==
+"@expo/image-utils@^0.2.5":
+  version "0.2.5"
+  resolved "https://registry.yarnpkg.com/@expo/image-utils/-/image-utils-0.2.5.tgz#dc7a5696fdbda5968cec6c8a7dcec1ac8bf69352"
+  integrity sha512-gG/3gwwOH5ETtBbRBoKOZIiEUAxy0xpGb6cmodEZycvzmiPTB+SjDbS/LqCMMAdvjWbA+9HuJhPX30PDqnas6g==
   dependencies:
     "@expo/spawn-async" "1.5.0"
     semver "6.1.1"
   optionalDependencies:
     sharp-cli "1.10.0"
 
+"@expo/json-file@8.1.4":
```

Comment:
Do you know why all these changes ended up in this PR?

Reply:
No, I don't know.